packages/database/src/models/PatientProgramRegistration.ts

```
@@ -18,10 +18,33 @@ export class PatientProgramRegistration extends Model {
   declare deactivatedClinicianId?: string;
   declare deactivatedDate?: string;
 
-  static initModel({ primaryKey, ...options }: InitOptions) {
+  static initModel(options: InitOptions) {
     super.init(
```

Biome (1.9.4)

[error] Using super in a static context can be confusing.

super refers to a parent class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)